src/main/java/com/coveo/pushapiclient/PlatformClient.java

```diff
@@ -82,17 +82,19 @@ public PlatformClient(String apiKey, String organizationId, Environment environm
      * Create a new push source
      *
      * @param name             The name of the source to create
+     * @param sourceType
      * @param sourceVisibility The security option that should be applied to the content of the source. See [Content Security](https://docs.coveo.com/en/1779).
      * @return
      * @throws IOException
      * @throws InterruptedException
      */
-    public HttpResponse<String> createSource(String name, SourceVisibility sourceVisibility) throws IOException, InterruptedException {
+    public HttpResponse<String> createSource(String name, final SourceType sourceType, SourceVisibility sourceVisibility) throws IOException, InterruptedException {
```

Comment:
Unless we don't export PlatformClient, this change is breaking. Is this a concern?

Reply:
yes I forgot 🤦 , 
@hdhayneCoveo can you tag the previous method with `@Deprecated` and let the users know they have to use the new method.
Once we do a major bump, we can get rid of the deprecated methods

Reply:
sounds good